### PR TITLE
Sort and remove duplicates from `.local.dic`

### PR DESCRIPTION
**Pull request by MrChocolatine**
Sort and remove duplicates from `.local.dic`.

### CODE REVIEW

File: .local.dic [hunk 1]
```diff
@@ -138,8 +141,8 @@ VoiceOver
 Vue
 W3C
 WAI-ARIA
-Watchman
 WCAG
+Watchman
```
Comment (BlueCutOfficial) [inline .local.dic:145]: I think this one used to be at the right position.

Comment (MrChocolatine) [reply .local.dic:145]: The sort and filter were done using JS, see the commits' body.

File: .local.dic [hunk 2]
```diff
@@ -335,17 +331,16 @@ lazy
...
 link
 lint
-linté
-lintés
-linter/nombrable
 linter/a0p+
+linter/nombrable
 linting
+linté
+lintés
```
[the remark this answers is not in the document]
Comment (MrChocolatine) [reply .local.dic:338]: The sort and filter were done using JS, see the commits' body.

[the remark this answers is not in the document]
Comment (MrChocolatine) [reply .local.dic]: New commit which sorts the file by taking into consideration non-ASCII characters.

File: .local.dic (hunk 1 above)
Comment (MrChocolatine) [reply .local.dic]: New commit which sorts the file by taking into consideration non-ASCII characters.

**Comment by BlueCutOfficial** [comment]
I prefer that version personally, it looks more readable to me and we can more easily spot duplications.

**Review by BlueCutOfficial — APPROVED** [review]

File: .local.dic [hunk 3]
```diff
@@ -297,56 +199,92 @@ geocode/GS
...
+IE
+IE11
 iframe/nombrable
 in-depth
 in-repo
+IndexedDB
+Inflector
 initialiseur/nombrable
 initializer/nombrable
+Initializer/nombrable
```
Comment (BlueCutOfficial) [inline .local.dic:243]: For instance, not sure the one with an uppercase `I` is still useful with `/nombrable`, we couldn't spot this specific kind of duplication with the initial sorting, so I prefer your new one.

**Pull request merged**